test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml

```diff
@@ -16,7 +16,8 @@ spec:
       serviceAccountName: csi-controller-sa
       containers:
         - name: csi-provisioner
-          image: gcr.io/gke-release/csi-provisioner:v1.2.1-gke.0
+          # TODO: replace with official 1.4.0 release when ready
+          image: quay.io/k8scsi/csi-provisioner:v1.4.0-rc1
```

Comment:
do we want to continue using the gcr image? (kickoff an rc1 release)

Reply:
I don't think we should release public rc images for gke.